docs/source/install.md

```diff
@@ -74,6 +74,58 @@ curl -sSL https://bit.ly/2ysbOFE | bash -s -- 2.4.2 1.5.2
   curl -sSL https://raw.githubusercontent.com/hyperledger/fabric/main/scripts/bootstrap.sh| bash -s 
   ```
 
+
+
+# New Fabric Install Script
+
+There is an updated install script with the same end-result as the existing script, but with an improved syntax. This script adopts the postitive opt-in approach to selecting the components to install. 
+
+To get the install script:
+
+```bash
+curl -sSLO https://raw.githubusercontent.com/hyperledger/fabric/main/scripts/install-fabric.sh && chmod +x install-fabric.sh
+```
+
+Run the script with the `-h` option to see the options
+
+```bash
+./install-fabric.sh -h
+Usage: ./install-fabric.sh [-f|--fabric-version <arg>] [-c|--ca-version <arg>] <comp-1> [<comp-2>] ... [<comp-n>] ...
```

Comment:
The only nit I would have is that I found this help text confusing. For example, this line :`d[ocker]|b[inary]|s[amples]` is very ugly and hard to parse.

Reply:
How about

```
<comp>: Component to install one or more of  docker | binary | samples | podman. (first letter of component also accepted)
```

Reply:
Much cleaner

Reply:
@nikhil550  thanks - I've updated the script accordingly